docs/application/native/guides/multimedia/media-playback.md

```diff
@@ -38,21 +38,17 @@ You can [play audio](#play_audio) and control functions for using audio resource
 
 The supported audio formats include MP3, AAC, WMA, M4A, 3GA, WAV, and AMR. The available formats depend on the target device.
 
-You can use multiple instances of the player to play several audio data streams concurrently. This means that your application can [play multiple uncompressed audio files](#play) at the same time.
+S/W and H/W audio codec could be selected by player API and the default codec type should be checked by calling player_get_audio_codec_type().
+
+You can use multiple instances of the player to play several audio data streams concurrently if S/W codec is selected. This means that your application can [play multiple uncompressed audio files](#play) at the same time.
 
 The following figure illustrates the general player state changes.
 
 **Figure: Player state changes**
 
 ![Player state changes](./media/using_players_state_changes.png)
 
-The `player_interrupted_cb()` callback is called when an application is interrupted by another application. The player is paused or moved to the IDLE state according to the audio session manager (ASM) policy. You must check the state of the current instance and take appropriate action. If the player is changed to the PAUSED state due to an interruption, the callback can be called at the end of the interruption with the `PLAYER_INTERRUPTED_COMPLETED` error code. For example, if the application is interrupted by a call, alarm, or notification, the completed message is posted. However, no messages are posted if the interruption has other causes. Make sure you handle the application accordingly when audio switching occurs between various applications.
-
-The following figure illustrates what happens when the player gets interrupted by the system.
-
-**Figure: Player states when interrupted by system**
-
-![Player states when interrupted by system](./media/using_players_interrupted_state.png)
+The `player_interrupted_cb()` callback could be called due to the resource restriction in the system. The player state will be one of IDLE, READY or PAUSED according to the resource management policy. You must check the state of the current instance and take appropriate action.
```

Comment:
Suggest rephrasing, please verify:

```suggestion
The `player_interrupted_cb()` callback could be called due to the resource restriction in the system. The player state could be IDLE, READY, or PAUSED based on the resource management policy. You must check the state of the current instance and take appropriate action.
```

Reply:
minor question. Is there any reason of removing "one of"?

Reply:
@surbhig12 Thanks for the description. If you fixed it because of the grammar issue, than please keep the original suggestion.